go.mod

```diff
@@ -54,20 +55,20 @@ require (
 	github.com/stretchr/objx v0.3.0 // indirect
 	github.com/stretchr/testify v1.7.1-0.20210116013205-6990a05d54c2 // includes ErrorContains
 	github.com/sykesm/zap-logfmt v0.0.2
-	github.com/syndtr/goleveldb v1.0.1-0.20210305035536-64b5b1c73954
-	github.com/tedsuo/ifrit v0.0.0-20180802180643-bea94bb476cc
+	github.com/syndtr/goleveldb v1.0.0
+	github.com/tedsuo/ifrit v0.0.0-20220120221754-dd274de71113
 	go.etcd.io/etcd v0.5.0-alpha.5.0.20181228115726-23731bf9ba55
 	go.uber.org/multierr v1.6.0 // indirect
 	go.uber.org/zap v1.16.0
 	golang.org/x/crypto v0.0.0-20210322153248-0c34fe9e7dc2
-	golang.org/x/tools v0.0.0-20200131233409-575de47986ce
+	golang.org/x/tools v0.0.0-20201224043029-2b0845dc783e
 	google.golang.org/grpc v1.31.0
 	gopkg.in/alecthomas/kingpin.v2 v2.2.6
 	gopkg.in/cheggaaa/pb.v1 v1.0.28
 	gopkg.in/yaml.v2 v2.4.0
 	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b // indirect
 )
 
-replace github.com/onsi/gomega => github.com/onsi/gomega v1.9.0
+replace github.com/golang/protobuf => github.com/golang/protobuf v1.3.3
```

Comment:
@Param-S , @denyeart , I am not sure if we can use replace here to make protobuf using version 1.3.3?

Reply:
```
yuanyi@yuanyideMacBook-Pro fabric % go mod why github.com/golang/protobuf                          
# github.com/golang/protobuf
(main module does not need package github.com/golang/protobuf)
```

Reply:
I think you need to wait until the Protobuf update from the other PR is merged.

Reply:
I see